operator/controllers/csiscaleoperator_controller.go

```diff
@@ -206,15 +206,11 @@ func (r *CSIScaleOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Req
 
 	logger.Info("Adding Finalizer")
 	if err := r.addFinalizerIfNotPresent(instance); err != nil {
-		message := "Couldn't add Finalizer"
+		message := "Failed to add finalizer"
```

Comment:
Please make these event message as informative as possible if possible, 

such as :

Failed to add `finalizer.csiscaleoperators.csi.ibm.com` to CSI Driver CustomResoruce instance named ibm-spectrum-scale-csi